src/actions/index.js

```diff
@@ -67,7 +67,7 @@ export const openFolder = (folderId, context = FILES_CONTEXT) => {
       context,
       folder: Object.assign(extractFileAttributes(folder), {
         parent: extractFileAttributes(parent)}),
-      files: folder.relations('contents').map(
+      files: (folder.relations('contents') || []).map(
```

Comment:
Is it the stack which defines what is `folder.relations('contents')` value? May we change the stack to send a empty array rather than `null` or `undefined`?